cpu/stm32/periph/rtc_all.c

```diff
@@ -257,10 +272,12 @@ void rtc_init(void)
 
     /* configure the EXTI channel, as RTC interrupts are routed through it.
      * Needs to be configured to trigger on rising edges. */
+    EXTI_REG_IMR  |= EXTI_IMR_BIT;
+#if !defined(CPU_FAM_STM32L5)
     EXTI_REG_FTSR &= ~(EXTI_FTSR_BIT);
     EXTI_REG_RTSR |= EXTI_RTSR_BIT;
-    EXTI_REG_IMR  |= EXTI_IMR_BIT;
     EXTI_REG_PR   = EXTI_PR_BIT;
+#endif
```

Comment:
I'm assuming the order does not matter here?

Reply:
I assume it does not. Because the `CPU_FAM_STM32L5` RTC interrupt line is not configurable (it's direct), the `FTSR`, `RTSR` and `PR` bits are not available. In `periph/rtt_all.c` it's implemented the same way.